src/components/graph/graph.config.js

```diff
@@ -96,6 +96,7 @@
  *   - 'wye'
  *
  * **[note]** react-d3-graph will map this values to [d3 symbols](https://github.com/d3/d3-shape#symbols)
+ * @param {function} [node.viewGenerator=undefined] - 🔍🔍🔍 function that receives a node and returns a JSX view.
```

Comment:
`{Function}`

Reply:
Can I define a different `viewGenerator` inside each node? If yes the 🔍🔍🔍 symbols are ok, otherwise remove them if you can only define one `viewGenerator` that will be used to render all nodes dynamically.

Reply:
Yeah! I don't know to which extent this will actually be useful (the same `viewGenerator` function could return different views depending on the circumstances, after all), but you can define a different one at node level. 👍